recipes/theano-pymc/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.0.5" %}
+
+package:
+  name: theano-pymc
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/T/Theano-PyMC/Theano-{{ version }}.tar.gz
+  sha256: bb566fa62f7923351299dfcb8947e42523057df65e81be444eb8ef0c6e602bf4
+  patches:
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```

Comment:
It looks like there is a python entry_point. Please add it here:

https://github.com/pymc-devs/Theano-PyMC/blob/master/setup.py#L131

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Reply:
Sorry, not quite sure what I need to do here. Could you provide a bit more info?

Reply:
```
build:
  number: 0
  entry_points:
    - theano-cache = bin.theano_cache:main
  script: {{ PYTHON }} -m pip install . -vv
```

Reply:
Done.